.github/workflows/backend_build_check.yml

```diff
@@ -0,0 +1,24 @@
+name: build database-service backend
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        os: [ubuntu-lastest]
+        node-version: [12.x] 
+
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
there is already a v2 version, but I don't know if this is relevant here.

Reply:
idk either, but v2 seems to work just as well, so i gonna switch to v2.